check_api/src/main/java/com/google/errorprone/fixes/SuggestedFix.java

```diff
@@ -68,8 +68,13 @@ public String toString(JCCompilationUnit compilationUnit) {
     StringBuilder result = new StringBuilder("replace ");
     for (Replacement replacement : getReplacements(compilationUnit.endPositions)) {
       result
-          .append("position " + replacement.startPosition() + ":" + replacement.endPosition())
-          .append(" with \"" + replacement.replaceWith() + "\" ");
+          .append("position ")
```

Comment:
@ronshapiro Good point! That's definitely readable, and that wouldn't report any warnings in my copy of IntelliJ, which was my main reason for submitting this PR (as I'm a huge stickler for these sorts of things :stuck_out_tongue_winking_eye:).